CommonTools/PileupAlgos/plugins/PuppiProducer.h

```diff
@@ -12,7 +12,10 @@
 #include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "DataFormats/Math/interface/PtEtaPhiMass.h"
-
+#include "DataFormats/Math/interface/LorentzVector.h"
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
```

Comment:
These three includes can be removed from PuppiProducer.cc, then

Reply:
Ok I'll check more careful what could be removed then